utils/gyb.py

```diff
@@ -2,13 +2,33 @@
 # GYB: Generate Your Boilerplate (improved names welcome; at least
 # this one's short).  See -h output for instructions
 
+from __future__ import print_function
+
 import re
-from cStringIO import StringIO
 import tokenize
 import textwrap
 from bisect import bisect
 import os
 
+import sys
+if sys.version_info < (3, 0):
+    from cStringIO import StringIO
+else:
+    from io import StringIO
+
+
+def next_meth(foo):
+    """
+    Returns the method-wrapper for foo.next() or foo.__next__(), depending
+    on the version of Python.  The builtin next() isn't suitable because
+    it calls the method.
+    """
+    if sys.version_info < (3, 0):
+        return foo.next
+    else:
+        return foo.__next__
```

Comment:
I believe the builtin Python function `next()` already implements this functionality.

- PEP for `next()`: https://www.python.org/dev/peps/pep-3114/
- Documentation for `next()`: https://docs.python.org/3/library/functions.html#next

Using the builtin, you could simply call `tokenize.generate_tokens(next(sourceLines.__iter__()))` below.